### DISCUSSION TITLE
Farming Simulator 25

### DISCUSSION THREAD

**Discussion by KR0311** [gameserverapp/Platform#968] [Game request]
Hey please can we add the newest addition to the most famous agriculture game

**Comment by mvaessen** [comment]
Hi, as far as I know are the farming simulator games limited to specific hosting parties. At least that is the case for older versions

**Comment by KR0311** [comment]
Technically yes however they do include tools to host a server withen the game files but  a couple of issues arise it looks like its windows only and you need to buy 2 copies of the game as they require you to own another license to host a server and a seperate one for playing the game too
